go/parquet/file/column_writer_test.go

```diff
@@ -222,6 +222,7 @@ type PrimitiveWriterTestSuite struct {
 	metadata   *metadata.ColumnChunkMetaDataBuilder
 	sink       *encoding.BufferWriter
 	readbuffer *memory.Buffer
+	//lint:ignore U1000 reader
 	reader     file.ColumnChunkReader
```

Comment:
if this isn't being used in the tests we can just remove this rather than having the lint ignore.